src/main/scala/co/datamechanics/delight/DelightStreamingConnector.scala

```diff
@@ -237,6 +238,7 @@ class DelightStreamingConnector(sparkConf: SparkConf) extends Logging {
       logWarning(s"Stopped waiting for pending events to be sent (max wait duration is ${maxWaitOnEnd}), although $nbPendingEvents were remaining")
     }
     sendAck()
+    logInfo(s"Application will be available in a few minutes on Delight at this url: $delightURL/apps/$dmAppId")
```

Comment:
```
logInfo(s"Application will be available a few minutes after it completes on Delight at this url: $delightURL/apps/$dmAppId")
```

Reply:
Sorry I thought this was at the beginning of the streaming, while it is at the end. Please discard!